package.json

```diff
@@ -119,6 +119,7 @@
   "dependencies": {
     "percentile": "^1.2.0",
     "react-bootstrap-table": "^4.3.1",
+    "react-bootstrap-modal":"^4.0.0",
```

Comment:
I'm going to remove that

Reply:
your going to remove the modal entirely? Because I couldn't install the plugin without adding that.